monitoring/defaults_linux.go

```diff
@@ -64,12 +64,18 @@ func DefaultProcessChecker() health.Checker {
 		"kubelet",
 		"planet",
 		"teleport",
-	}}
+	}
+
+	if openEBSEnabled {
+		processes = append(processes, "iscsid")
+	}
+
+	return &ProcessChecker{processes}
 }
 
 // BasicCheckers detects common problems preventing k8s cluster from
 // functioning properly
-func BasicCheckers(checkers ...health.Checker) health.Checker {
+func BasicCheckers(openEBSEnabled bool, checkers ...health.Checker) health.Checker {
```

Comment:
Same comment as above, make it agnostic to OpenEBS, just make the public API more flexible.